### PR TITLE
Improved vehicle picker

### PR DESCRIPTION
**Pull request by DylanVanAssche**
Finally improved the vehicle picker. Uses regex matching to figure out the right file.
Now all the requests should return the reliability graph.

Also fixed the reliability transfer time and added missing data for some trains.

**Note**: Print statements are added for faster debugging.

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -14,4 +14,4 @@ requests = "*"
 xlsx2csv = "*"
 
 [requires]
-python_version = "3.7"
+python_version = "3.6"
```
Comment (juliavdkris) [inline Pipfile:17]: Is there a reason for the Python 3.6 downgrade? As far as I know Python 3.7 doesn't really add any functionality that we need, but I was just wondering.
Also, 3.8 was recently released and it adds some nice stuff like a walrus operator and a `=` sign in f-strings (prints more information on a variable, nice for debugging).

File: connections.py [hunk 2]
```diff
@@ -7,6 +7,7 @@
 import time as tm
 import json
 import re
+import glob
 import os.path
```
Comment (juliavdkris) [inline connections.py:11]: Not needed anymore since we're using glob now

File: connections.py [hunk 3]
```diff
@@ -129,23 +131,27 @@ def handle_invalid_vehicle_id(self, vehicle_id):
...
+                    print(arr_reliability)
 
                 if 'vias' in connection:
                     for via in connection['vias']['via']:
                         via_station = via['stationinfo']['@id']
-                        dep_vehicle_id = self.handle_invalid_vehicle_id(via['departure']['vehicle'].split('.')[-1])
-                        arr_vehicle_id = self.handle_invalid_vehicle_id(via['arrival']['vehicle'].split('.')[-1])
+                        dep_vehicle_id = re.findall(r'\d+', via['departure']['vehicle'])[0]  #self.handle_invalid_vehicle_id(connection['departure']['vehicle'].split('.')[-1])
+                        arr_vehicle_id = re.findall(r'\d+', via['arrival']['vehicle'])[0] #self.handle_invalid_vehicle_id(connection['arrival']['vehicle'].split('.')[-1])
+                        print("\t{} {}".format(arr_vehicle_id, dep_vehicle_id))
```
Comment (juliavdkris) [inline connections.py]: Excuse me sir, do you have a moment to talk about our lord and savior, the f-string?

```python
print(f'\t{arr_vehicle_id} {dep_vehicle_id}')
```

**Review by juliavdkris — APPROVED** [review]
Nice!

I had a few questions, mostly about the Python 3.6 downgrade (the rest is just nitpicking)

File: Pipfile (hunk 1 above)
Comment (DylanVanAssche) [reply Pipfile:17]: Ubuntu 18.04 LTS is still using Python 3.6 :(

File: connections.py (hunk 2 above)
Comment (DylanVanAssche) [reply connections.py:11]: Used to check if the file exists

**Pull request merged**